sentinel/src/process-hooks.js

```diff
@@ -0,0 +1,13 @@
+const feed = require('./lib/feed');
+const schedule = require('./schedule');
+
+module.exports = {
+  init: () => {
+    process.on('SIGUSR1', () => {
```

Comment:
I struggled to find any documentation on this but I think we can call the signals whatever we want, right? If so let's use clearer names. What about `SIGLISTEN`, `SIGSTOPLISTENING`, and `SIGRUNTASKS`? Or even `SIGSENTINELLISTEN` etc? It's getting long but at least it'll be clear that it's a custom signal and not a Node thing?

Also I think `SIGUSR1` is reserved and according to this documentation will trigger Node into debugging mode: https://nodejs.org/en/learn/getting-started/debugging - so if we can't rename as above, let's use a different number at least.

Reply:
Unfortunately, you can't send custom signals: 
```
➜  ~ kill -S SIGSIGSTOPLISTENING 26360
kill: unknown signal: SIGS
kill: type kill -l for a list of signals
```

So you need to select one of the signals in the list. There are some random ones that might not kill the process, but most do, and it felt weird to send a `SIGURG` to stop feed listening. 
This is why there's a toggle to stop/start processing, because we can't send custom signals and there are only two which are "user customizable" - SIGUSR1 and SIGUSR2, which I am using.

Reply:
Ugh that's disgusting. Ok. The only other way I can see to do it would be to have a single signal which then looks somewhere else (file or db) to reload config, but that almost defeats the purpose!

Let's go with it for now.